packages/communicator-bcrypt/package.json

```diff
@@ -20,7 +20,7 @@
     "@types/bcryptjs": "^2.4.6",
     "grunt": "^1.6.1",
     "nerdbank-gitversioning": "^3.6.133",
-    "typescript": "^5.3.3"
+    "typescript": "5.4.5"
```

Comment:
Is it deliberate to drop `^`?

Reply:
no, will fix. Should I also add it back for the main package.json?